templates/CRM/Pledge/Form/Pledge.tpl

```diff
@@ -144,6 +149,13 @@
 // bind first click of accordion header to load crm-accordion-body with snippet
 // everything else taken care of by cj().crm-accordions()
 cj(document).ready( function() {
+
+  cj('#currency').closest('tr').next('tr').find('.currency-symbol').text(cj('#currency option:selected').text());
+  // if there are more than one currency enabled.
+   cj('#currency').change(function(){
+     cj('#currency').closest('tr').next('tr').find('.currency-symbol').text(cj(this).find(':selected').text());
```

Comment:
I'm changing whole text instead of just currency symbol as many currencies does not have symbol, let me know if you think other way.